flat-manager/index.js

```diff
@@ -12,13 +12,25 @@ const run = (repository, flatManagerUrl, token) => {
   ])
     .then(async () => {
       let buildId = ''
-      const exitCode = await exec.exec('flat-manager-client', [
+      let args = [
         '--token',
-        token,
+        token        
+      ]
+
+      if (end_of_life && end_of_life_rebase) {
```

Comment:
you can have an end-of-life without having the later but not the opposite iirc.

Reply:
never mind, i was looking at an older commit

Reply:
Was perhaps a little too early in the morning :sweat_smile: